src/integrationTest/groovy/wooga/gradle/xcodebuild/IntegrationSpec.groovy

```diff
@@ -26,22 +26,8 @@ import org.junit.contrib.java.lang.system.EnvironmentVariables
 import org.junit.contrib.java.lang.system.ProvideSystemProperty
 import spock.lang.Requires
 
-@Requires({ os.macOs })
-class IntegrationSpec extends nebula.test.IntegrationSpec {
-
-    @Rule
-    ProvideSystemProperty properties = new ProvideSystemProperty("ignoreDeprecations", "true")
-
-    @Rule
-    public final EnvironmentVariables environmentVariables = new EnvironmentVariables()
-
-    def escapedPath(String path) {
-        String osName = System.getProperty("os.name").toLowerCase()
-        if (osName.contains("windows")) {
-            return StringEscapeUtils.escapeJava(path)
-        }
-        path
-    }
+//@Requires({ os.macOs })
```

Comment:
I assume we don't need the blanket `@Requires` anymore here?

Reply:
No, not anymore.